.tool-versions

```diff
@@ -0,0 +1,2 @@
+gradle 6.5
+java adopt-openjdk-11+28
```

Comment:
```suggestion
java adoptopenjdk-11.0.0+28
```

Reply:
Huh, is that what asdf suggested for you?

Reply:
Right that's what appears for me on `asdf list-all java` as well for @ecomba. Perhaps you have a diff `asdf-java` plugin installed?